vm/src/protocol/number.rs

```diff
@@ -216,6 +218,97 @@ impl PyNumberMethods {
     };
 }
 
+pub enum PyNumberMethodsOffset {
```

Comment:
Name suggestion:
```suggestion
pub enum PyNumberBinaryOp {
```

I don't think we need a single linear enum to represent both binary and unary ops.